README.md

```diff
@@ -1 +1,2 @@
-# skyux-tree-view
\ No newline at end of file
+# skyux-tree-view
+SKYUX styles and utilities to be used with angular-tree-view.
```

Comment:
Needs the "shields" for NPM, coverage, and Travis-CI build results.

Reply:
Done.